tests/test_openml/test_config.py

```diff
@@ -175,13 +175,14 @@ def test_configuration_file_not_overwritten_on_load():
 
 def test_configuration_loads_booleans(tmp_path):
     config_file_content = "avoid_duplicate_runs=true\nshow_progress=false"
-    with (tmp_path / "config").open("w") as config_file:
+    tmp_file = tmp_path / "config"
+    with tmp_file.open("w") as config_file:
         config_file.write(config_file_content)
-        read_config = openml.config._parse_config(tmp_path)
+    read_config = openml.config._parse_config(tmp_file)
```

Comment:
This was passed a directory... which lead to a silent error and an empty config, which went unnoticed because the values below happened to coincide with the old defaults :D